### PR TITLE
fix: remove the info icon popover from the toolbar

### PR DESCRIPTION
**Pull request by just-toby**
## Description

removes the info icon tooltip from the toolbar - this is in accordance with the new designs mentioned in the jira ticket.

_JIRA ticket:_ https://uniswaplabs.atlassian.net/browse/WEB-3239
_Slack thread:_ https://uniswapteam.slack.com/archives/C047U65H422/p1682007265763589

## Screen capture

| Before       | After (Desktop) | After (Mobile) |
| ------------ |---------------- | -------------- |
| image | image      | image    |

### Reproducing the error

1.  load a swap quote and verify that the icon is misaligned

### QA (ie manual testing)

- [x] load a swap quote and verify that the icon is gone, and the text is aligned correctly

### Automated testing

- [x] <s>Unit test</s>
- [x] <s>Integration/E2E test</s>

### CODE REVIEW

**Comment by JFrankfurt** [comment]
is there some to-be-added mobile-specific portion of this pr, or just the screenshot missing? 
image

File: src/components/swap/SwapDetailsDropdown.tsx [hunk 1]
```diff
@@ -128,36 +119,14 @@ export default function SwapDetailsDropdown({ trade, syncing, loading, allowedSl
           shouldLogImpression={!showDetails}
         >
           <StyledHeaderRow onClick={() => setShowDetails(!showDetails)} disabled={!trade} open={showDetails}>
-            <RowFixed style={{ position: 'relative' }}>
+            <RowFixed style={{ position: 'relative' }} align="center">
               {loading || syncing ? (
```
Comment (JFrankfurt) [inline src/components/swap/SwapDetailsDropdown.tsx:123]: `Boolean(loading || syncing) && (`

Comment (JFrankfurt) [reply src/components/swap/SwapDetailsDropdown.tsx:123]: so you can drop the null case

**Review by grabbou — APPROVED** [review]
Approving with Jordans request for wrapping in a `Boolean`.

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**